### PR TITLE
Properties returning collection trivia with count equal to 1

### PR DESCRIPTION
**Pull request by evnik**

### CODE REVIEW

File: Sources/SwiftSyntax/Trivia.swift.gyb [hunk 1]
```diff
@@ -99,19 +99,24 @@ public struct Trivia {
...
-    /// Return a piece of trivia for some number of '${joined}' characters.
-    public static func ${trivia.lower_name}s(_ count: Int) -> Trivia {
-      return [.${trivia.lower_name}s(count)]
-    }
+  /// Returns a piece of trivia for some number of '${joined}' characters.
+  public static func ${trivia.lower_name}s(_ count: Int) -> Trivia {
+    return [.${trivia.lower_name}s(count)]
+  }
+
+  /// Gets a piece of trivia for '${joined}' characters.
```
Comment (ahoppen) [inline Sources/SwiftSyntax/Trivia.swift.gyb:111]: I think the current comment isn’t grammatically correct.

```suggestion
  /// Gets a piece of trivia with a single '${joined}' character.
```

File: Sources/SwiftSyntax/Trivia.swift.gyb [hunk 2]
```diff
@@ -99,19 +99,24 @@ public struct Trivia {
...
+  public static var ${trivia.lower_name}: Trivia {
+    return [.${trivia.lower_name}s(1)]
```
Comment (ahoppen) [inline Sources/SwiftSyntax/Trivia.swift.gyb:113]: I don’t even think we need the `[` and `]`. Without those we’re just calling into the function defined above.

File: Tests/SwiftSyntaxBuilderTest/VariableTests.swift [hunk 3]
```diff
@@ -30,15 +30,13 @@ final class VariableTests: XCTestCase {
...
-        for i in 1...3 {
-          ArrayElement(
-            expression: IntegerLiteralExpr(i),
-            trailingComma: .comma.withoutTrailingTrivia().withTrailingTrivia(.newlines(1)))
-        }
-      })))
+      PatternBinding(pattern: "a", initializer: InitializerClause(
+        value: ArrayExpr(leftSquare: .`leftSquareBracket`.withTrailingTrivia(.newline)) {
+          for i in 1...3 {
+            ArrayElement(expression: "\(i)", trailingComma: .comma.withoutTrailingTrivia().withTrailingTrivia(.newline))
```
Comment (ahoppen) [inline Tests/SwiftSyntaxBuilderTest/VariableTests.swift:36]: I don’t think this transformation is technically correct. The old example generates an `IntegerLiteralExpr` while this creates an `IdentifierExpr` (which conforms to `ExpressibleAsIdentifierExpr`). Identifiers are e.g. variable names, which shouldn’t consist purely of digits. 

While both generate the same source code, the SwiftSyntax tree with the `IdentifierExpr` shouldn’t technically be valid.

**Review by ahoppen — COMMENTED** [review]
Thanks. This looks good. I’ve got a couple nitpicks inline.

**Comment by kimdv** [comment]
@swift-ci please test

**Comment by ahoppen** [comment]
@swift-ci Please test

**Pull request merged**